client/me/security-2fa-key/index.jsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons'; // eslint-disable-line no-restricted-imports
+import React from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import Security2faKeyAdd from './add';
+import Security2faKeyList from './list';
+import { recordGoogleEvent } from 'state/analytics/actions';
+import { isWebauthnSupported } from 'lib/webauthn';
+import wpcom from 'lib/wp';
+import Notice from 'components/notice';
+
+class Security2faKey extends React.Component {
+	static initialState = Object.freeze( {
+		addingKey: false,
+		isBrowserSupported: true,
+		errorMessage: false,
+		security2faChallenge: {},
+		security2faKeys: [],
+	} );
+
+	state = this.constructor.initialState;
+
+	componentDidMount = () => {
+		this.getKeysFromServer();
+		isWebauthnSupported().then( this.setSupported );
+	};
+
+	getClickHandler = ( action, callback ) => {
+		return event => {
+			this.props.recordGoogleEvent( 'Me', 'Clicked on ' + action );
+
+			if ( callback ) {
+				callback( event );
+			}
+		};
+	};
+
+	setSupported = supported => {
+		this.setState( { isBrowserSupported: supported } );
+	};
+
+	addKeyStart = event => {
+		event.preventDefault();
+		this.setState( { addingKey: true } );
+	};
+
+	addKeyRegister = () => {
+		this.getKeysFromServer();
+	};
+
+	deleteKeyRegister = keyData => {
+		console.log( 'Register key: ', keyData ); //eslint-disable-line
+		wpcom.req.get(
+			'/me/two-step/security-key/delete',
+			{ credential_id: keyData.id },
+			this.getKeysFromServer
+		);
+	};
+
+	addKeyCancel = () => {
+		this.setState( { addingKey: false } );
+	};
+
+	keysFromServer = ( err, data ) => {
+		this.setState( { addingKey: false, security2faKeys: get( data, 'registrations', [] ) } );
+	};
+
+	getChallenge = () => {
+		wpcom.req.get( '/me/two-step/security-key/registration_challenge', {}, this.setChallenge );
+	};
+
+	setChallenge = ( error, data ) => {
+		this.setState( { security2faChallenge: data } );
+	};
+
+	getKeysFromServer = () => {
+		wpcom.req.get( '/me/two-step/security-key/get', {}, this.keysFromServer );
+	};
+
+	render() {
+		const { translate } = this.props;
+		const { addingKey, isBrowserSupported, errorMessage, security2faKeys } = this.state;
+		return (
+			<div className="security-2fa-key">
+				<SectionHeader label={ translate( 'Security Key' ) }>
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Security Key Added' )` *ES Score: 6*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/gm19-webauthn--android-support)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).